common-util/Login/config.jsx

```diff
@@ -19,37 +15,33 @@ export const SUPPORTED_CHAINS = (
 
 export const projectId = process.env.NEXT_PUBLIC_WALLET_PROJECT_ID;
 
-const { publicClient, webSocketPublicClient, chains } = configureChains(
-  SUPPORTED_CHAINS,
-  [
-    jsonRpcProvider({
-      rpc: (chain) => ({
-        http: RPC_URLS[chain.id],
-      }),
-    }),
-    w3mProvider({ projectId }),
-  ],
-);
+const metadata = {
+  name: SITE_TITLE,
+  description: SITE_DESCRIPTION,
+  url: SITE_URL,
+  icons: ['https://avatars.githubusercontent.com/u/37784886'],
+};
 
-export const wagmiConfig = createConfig({
-  autoConnect: true,
-  logger: { warn: null },
-  connectors: [
-    ...w3mConnectors({
-      projectId,
-      version: 2, // v2 of wallet connect
-      chains,
-    }),
-    new SafeConnector({
-      chains,
-      options: {
-        allowedDomains: [/gnosis-safe.io$/, /app.safe.global$/],
-        debug: false,
-      },
-    }),
-  ],
-  publicClient,
-  webSocketPublicClient,
+/**
+ * @type {import('@web3modal/wagmi').WagmiOptions}
+ */
+export const wagmiConfig = defaultWagmiConfig({
+  chains: SUPPORTED_CHAINS,
+  projectId,
+  metadata,
+  ssr: true,
+  storage: createStorage({ storage: cookieStorage }),
 });
 
-export const ethereumClient = new EthereumClient(wagmiConfig, chains);
+if (!projectId) throw new Error('Project ID is not defined');
+
+createWeb3Modal({
```

Comment:
same q as for member app - do you think it'll make more sense to move it to app?

Reply:
yes, referring to `_app`, it's just for clarity -  atm this function runs just when you import any other constant from this file - this may be unclear